### ISSUE TITLE
Change the color of ️✔️ from grey to green.

### ISSUE THREAD

**Issue by MyriaCore** [twitter/twemoji#426]
### TL;DR

Change the color of :heavy_check_mark: from grey to green.

### The Problem

`heavy_check_mark`, or ✔, is currently a dark grey color. This may be problematic because it can be hard to see on dark backgrounds (touched upon in #420), but more importantly for the scope of *this* issue, it's more difficult to visually identify it as a counterpart for :x:, which is red:

<table>
<th>Twemoji</th>
<td><img height=72 width=72 src="https://user-images.githubusercontent.com/35076430/88485114-9baf4400-cf41-11ea-8c8a-9201c5e6be71.png"></td>
<td><img height=72 width=72 src="https://user-images.githubusercontent.com/35076430/88485317-ff863c80-cf42-11ea-80ec-21cb7de32e07.png"></td>
</table>

While I'm pretty sure :heavy_check_mark: and :x: aren't *officially* counterparts, they're most often used that way by emoji users. If :x: was also the same dark-grey as :heavy_check_mark:, it might make more sense (since it'd be a bit more self-consistent). However, making both grey would lower the contrast between :x: and :heavy_check_mark:, which *also* makes it harder to visually identify them as counterparts. 

### The Solution

So, it appears the best solution to this is to make :heavy_check_mark: green, so it is easier to visually identify as a contrasting counterpart to :x:. This solution is also proposed to resolve #420, and it's what Microsoft and what GitHub appear to use:

<table>
<tr>
<th>Microsoft</th>
<td><img height=72 width=72 src="https://user-images.githubusercontent.com/35076430/88485492-2133f380-cf44-11ea-9847-67fc30b66daa.png"></td>
<td><img height=72 width=72 src="https://user-images.githubusercontent.com/35076430/88485503-2e50e280-cf44-11ea-9c17-33268c31f06b.png"></td>
</tr>

<tr>
<th>GitHub</th>
<td><img height=72 width=72 src="https://github.githubassets.com/images/icons/emoji/unicode/2714.png"></td>
<td><img height=72 width=72 src="https://github.githubassets.com/images/icons/emoji/unicode/274c.png"></td>
</table>

### Why it matters

This is important because at the moment, there's no substitute for :heavy_check_mark: and :x: that *are* visually contrasting. The closest possible pair is the check/cross mark button emojis, :white_check_mark:/:negative_squared_cross_mark:, and as you can see, the colors for both of these are *exactly* the same, making it harder to visually identify these two at a glance (as discussed in #285): 

<table>
<th>Twemoji</th>
<td><img height=72 width=72 src="https://user-images.githubusercontent.com/35076430/88485686-b388c700-cf45-11ea-9733-8126bb29954c.png"></td>
<td><img height=72 width=72 src="https://user-images.githubusercontent.com/35076430/88485675-a5d34180-cf45-11ea-8ca8-263896e9c26c.png"></td>
</table>

That means that if #285 stays closed and this stays unadressed, twemoji users who would like easily distinguishable check/cross mark emojis have **no workaround**, and would be forced to either switch from twemoji, or deal with having counterparts that are difficult to distinguish.

**Comment by AbyssStudios** [comment]
I made pull request #439 to solve this issue.

**Comment by jdecked** [comment]
We'd be an outlier with an overwhelming majority of emoji providers if we changed, so at the moment this isn't something we plan to do.
